File: src/main/java/calebzhou/rdi/microservice/model/json/weather/Content.java
package calebzhou.rdi.microservice.model.json.weather;

public class Content {
    private String source;
    private String title;
    private String description;

    public String getSource() {
        return source;
    }

    public Content setSource(String source) {
        this.source = source;
        return this;
    }

    public String getTitle() {
        return title;
    }

    public Content setTitle(String title) {
        this.title = title;
        return this;
    }

    public String getDescription() {
        return description;
    }

    public Content setDescription(String description) {
        this.description = description;
        return this;
    }
}

File: src/main/kotlin/calebzhou/rdi/microservice/model/json/weather/Content.kt
package calebzhou.rdi.microservice.model.json.weather

import com.google.gson.annotations.SerializedName


data class Content (

  @SerializedName("province"       ) var province      : String?           = null,
  @SerializedName("status"         ) var status        : String?           = null,
  @SerializedName("code"           ) var code          : String?           = null,
  @SerializedName("description"    ) var description   : String?           = null,
  @SerializedName("regionId"       ) var regionId      : String?           = null,
  @SerializedName("county"         ) var county        : String?           = null,
  @SerializedName("pubtimestamp"   ) var pubtimestamp  : Double?              = null,
  @SerializedName("latlon"         ) var latlon        : ArrayList<Double> = arrayListOf(),
  @SerializedName("city"           ) var city          : String?           = null,
  @SerializedName("alertId"        ) var alertId       : String?           = null,
  @SerializedName("title"          ) var title         : String?           = null,
  @SerializedName("adcode"         ) var adcode        : String?           = null,
  @SerializedName("source"         ) var source        : String?           = null,
  @SerializedName("location"       ) var location      : String?           = null,
  @SerializedName("request_status" ) var requestStatus : String?           = null

)